work15/index.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+const modalBtn = document.getElementById("js_modalButton");
+const modal = document.getElementById("js_modal");
+const requestBtn = document.getElementById("js_requestButton");
+
+const url = "https://mocki.io/v1/d6da0b8a-3546-419e-aaec-05f3a247c6d0";
+// 間違っているURLの場合↓
+// const url = "https://myjson.dit.upm.es/api/bins/ほげほげajy3";
+// 空の配列の場合↓
+// const url = "https://mocki.io/v1/242b685f-a3d7-45a8-aeca-0376bd495b89";
+// 404エラーの場合↓
+// const url = "https://httpstat.us/404";
+
+// 下記は、myjson繋がらない時の固定値
+// const url = {
+//   "data": [
+//     {
+//       "a": "bookmark",
+//       "img": "img/1.png",
+//       "alt": "画像１",
+//       "text": "ブックマーク"
+//     },
+//     {
+//       "a": "message",
+//       "img": "img/2.png",
+//       "alt": "画像２",
+//       "text": "メッセージ"
+//     }
+//   ]
+// }
+
+async function getData() {
+  try {
+    const response = await fetch(url);
+    if (response.ok) {
+      const json = await response.json();
+      return json;
+    } else {
+      console.error(`${response.status}:${response.statusText}`);
+    }
+    // 下記は、固定値をそのままpromiseの返り値とする
+    // const json = url
+    // return json
```

Comment:
Q. Is this comment necessary? 
I didn't think we should keep too much that is not needed for implementation.

Reply:
I've left comments on my assignments so far, but I should have only displayed them when needed.

thank you.

[06a87ba](https://github.com/yasu-pro/js-handson/pull/17/commits/06a87ba39f496c987eef99979e5bebf6d0228f57)